src/main/java/com/dev/cinema/dto/ShoppingCartRequestDto.java

```diff
@@ -0,0 +1,22 @@
+package com.dev.cinema.dto;
+
+public class ShoppingCartRequestDto {
+    private Long urerId;
```

Comment:
```suggestion
    private Long userId;
```
уважніше, будь ласка